source/transactions-convenient-api/transactions-convenient-api.rst

```diff
@@ -299,6 +299,18 @@ MongoClient within the callback in order to execute operations within the
 transaction. Per the `Driver Session`_ specification, ClientSessions should
 already provide access to a client object.
 
+Errors handling inside the callback
+-----------------------------------
+
+Drivers MUST document that the callback MUST NOT silently swallow errors that
+do not have the "TransientTransactionError" label. Such errors abort
```

Comment:
Is it guaranteed that the transaction will be aborted? Also, consider adding an example error. DuplicateKeyError would be the obvious choice since that's what prompted the clarification.

Reply:
> Is it guaranteed that the transaction will be aborted?
Looks like this is not stated anywhere, so I changed the wording to "may abort the transaction". But I thinks we should play safe here.

> Also, consider adding an example error. 
Added an example with some additional clarification.